OtripleS.Web.Api.Tests.Acceptance/Brokers/OtripleSApiBroker.Calenders.cs

```diff
@@ -19,7 +19,8 @@ public partial class OtripleSApiBroker
 
         public async ValueTask<Calendar> GetCalenderByIdAsync(Guid calenderId) =>
             await this.apiFactoryClient.GetContentAsync<Calendar>($"{CalendarRelativeUrl}/{calenderId}");
-
+        public async ValueTask<Calendar> PutCalendarAsync(Calendar calendar) =>
+         await this.apiFactoryClient.PutContentAsync(CalendarRelativeUrl, calendar);
```

Comment:
This needs a tab.
Make sure you separate multi-liners with a new line between them and any neighboring functions/methods.